src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
+
+    public ActorServiceImpl() {
+        actorDao = new ActorDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
Consider passing the `SessionFactory` as a parameter to the constructor of `ActorServiceImpl` instead of directly accessing `HibernateUtil.getSessionFactory()`. This approach improves testability and decouples the service from the utility class.